mimic/model/cloudfeeds.py

```diff
@@ -0,0 +1,86 @@
+"""
+Emulate the behavior of Rackspace Cloud Feeds service.
```

Comment:
Non-blocking - could we include a link to the API docs (I assume http://docs.rackspace.com/cloud-feeds/api/v1.0/feeds-devguide/content/overview.html?)